### PR TITLE
Specify container when running kubectl logs

### PR DESCRIPTION
**Pull request by chxchx**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes https://github.com/istio/istio/issues/970

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: tests/e2e/framework/testInfo.go [hunk 1]
```diff
@@ -150,7 +150,8 @@ func (t testInfo) FetchAndSaveClusterLogs(namespace string) error {
 					glog.Warningf("Error during closing file: %v\n", err)
 				}
 			}()
-			dump, err := util.ShellMuteOutput(fmt.Sprintf("kubectl logs %s -n %s", pod, namespace))
+			dump, err := util.ShellMuteOutput(
```
Comment (gyliu513) [inline tests/e2e/framework/testInfo.go]: The commit message needs to be updated to 

```
Specify container when running kubectl logs
```

Should be `container` but not `cluster`

Comment (chxchx) [reply tests/e2e/framework/testInfo.go]: Yes I notice that typo assume as I created this PR and correct it. Ditto for the PR title.

**Comment by ldemailly** [comment]
Plz add Fixes... the issue I filled, thx!

**Comment by chxchx** [comment]
@ldemailly Do you mind sharing a link to the issue you filed? Thanks!

**Comment by ldemailly** [comment]
you have it already #970

**Pull request merged**